sherpa/conftest.py

```diff
@@ -82,6 +82,7 @@ def pytest_configure(config):
     """
     try:
         path = config.getoption(TEST_DATA_OPTION)
-        SherpaTestCase.datadir = path
+        if (path):
```

Comment:
You are right, I switch context too often between JVM languages and Python, and this is the result :)